File: modules/x-engine-native-localstorage__webproxy/iOS/Class/Native_localstorage__webproxy.m
//
//  Native_localstorage__webproxy.m
//  localstorage__webproxy
//


#import "Native_localstorage__webproxy.h"
#import "XENativeContext.h"

@interface Native_localstorage__webproxy()
{ }
@end

@implementation Native_localstorage__webproxy
NATIVE_MODULE(Native_localstorage__webproxy)

 - (NSString*) moduleId{
    return @"com.zkty.native.localstorage__webproxy";
}

- (int) order{
    return 0;
}

- (void)afterAllNativeModuleInited{
} 
-(NSString*) test{
    return @"test";
}
@end
 

File: modules/x-engine-native-localstorage__webproxy/android/native-localstorage__webproxy/src/main/java/com/zkty/nativ/localstorage__webproxy/Nativelocalstorage__webproxy.java
package com.zkty.nativ.localstorage__webproxy;

import android.util.Log;

import com.tencent.mmkv.MMKV;
import com.zkty.nativ.core.NativeModule;
import com.zkty.nativ.core.XEngineApplication;

import java.io.File;

public class Nativelocalstorage__webproxy extends NativeModule implements Ilocalstorage__webproxy {

    @Override
    public String moduleId() {
        return "com.zkty.native.localstorage__webproxy";
    }

    @Override
    public void afterAllNativeModuleInited() {

    }
}
